src/int/expand.rs

```diff
@@ -0,0 +1,17 @@
+use crate::{ConstantTimeSelect, Int, Uint};
+
+impl<const LIMBS: usize> Int<LIMBS> {
+    /// Expand the representation of `self` to an `Int<T>`.
+    /// Assumes `T >= LIMBS`; panics otherwise.
+    #[inline(always)]
+    pub fn expand<const T: usize>(&self) -> Int<T> {
```

Comment:
Uint has resize, maybe we can keep consistency and name it the same?

Reply:
When developing, I initially used `resize` as well. However, it felt weird to downsize an `Int`, knowing that you could be losing sign information :thinking: 

But then again, I suppose the user should know that this is an "unsafe" operation that could lead to data loss :man_shrugging: 

I'll make it happen.

Reply:
you should not be losing sign information, resize(Int) is not the same as Int(resize(Uint)).
e.g. resize 8-bit `-1` which is `11111111` is perfectly fine resizable to 4 bits as `1111` which is still `-1`, because `-1` has only one bit of information.
In summary when -2^k <= x < 2^k, then you should be able to downsize Int to k-bits without losing it's value (without overflowing or whatever we call it).